kura/org.eclipse.kura.core.net/src/main/java/org/eclipse/kura/core/net/WifiAccessPointImpl.java

```diff
@@ -141,4 +142,82 @@ public String toString() {
         }
         return sb.toString();
     }
+    
+    @Override
+	public int hashCode() {
```

Comment:
To save verbosity, you could use `Objects.hash(Object...)` and `Objects.equals(Object, Object)`